scripts/run-node.py

```diff
@@ -8,21 +8,28 @@
 import requests
 import json
 import zipfile
+import hashlib
+import math
 from io import BytesIO
 
 # Set up logging
 logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
 
 # Advanced user configs
-moniker = "pynode"  # Custom moniker for the node
-trust_height_delta = 20000  # Negative height offset for state sync
-enable_unsafe_reset = True  # Wipe database and keys before setup
+moniker = "seinode"  # Custom moniker for the node
+trust_height_delta = 0  # Negative height offset for state sync
 version_override = False  # Override version fetching. if true, specify version(s) below
+p2p_port = 26656
+rpc_port = 26657
+lcd_port = 1317
+grpc_port = 9090
+grpc_web_port = 9091
+pprof_port = 6060
 
 # Chain binary version ["version_override" must be true to use]
-MAINNET_VERSION = "v3.9.0"
-DEVNET_VERSION = "v5.2.2"
-TESTNET_VERSION = "v5.2.2"
+MAINNET_VERSION = "v5.9.0-hotfix"
```

Comment:
Yes, it's automatically pulling the latest version tag from the rpc endpoint already. This is only for overriding the version

Reply:
Yeah these version numbers are only for manual override, otherwise it should fetch the current version from abci_info and download that binary (if available).
Maybe we should have a fallback that will build it from source if the prebuilt bin is not available.

Reply:
Yes, it's automatically pulling the latest version tag from the rpc endpoint already. This is only for overriding the version

Reply:
@cordt-sei good idea, i'll add this

Reply:
the only thing i'd suggest is try to get it from abci_info, if it fails instead ask the owner to manually enter - i would prefer this just because it's better than defaulting to a version that doesn't work